sql-mutations/diplomat.sql

```diff
@@ -0,0 +1,2 @@
+delete from "addresses"
+where "addressId" = '161'
```

Comment:
Remember that if we are checking if a value is a number, we should drop the quotes. It still works, because Postgres does type coercion, but for consistency and clarity, it's better to drop the quotes